README.md

```diff
@@ -106,35 +113,52 @@ An example of a configuration file can be found in [test_config.ini](https://git
 
 After processing the configuration file, DaLI reads crypto data from native sources and generates a RP2 input ODS file and a RP2 configuration file in the `output` directory or where specified with the `-o` CLI option.
 
+Provided DaLI is installed, to generate RP2 input files, open a terminal window (or PowerShell if on Windows).
+
+Create the directory structure and download the example input files.
+
 To try DaLI with the example configuration, download the following files:
```

Comment:
I suggest a structure like the following:
If your system has wget:
  <wget-based instructions>
If your system doesn't have wget:
 <manual instructions>

Generate output files...